### DISCUSSION TITLE
PINN Configuration

### DISCUSSION THREAD

**Discussion by dshuman52** [procount/pinn#854] [Q&A] [answered]
I am old enough to be senile, I may have been hallucinating, and I don't drink.  Over a week of using PINN I believe I saw windows (not config file editing instructions) to update the PINN configuration where I learned I probably wanted vncshare as an option and made a note to go back and check out the rest of the options.  I have since found the config file documentation but I cannot seem to find the windows with checkboxes to update the configuration.  Is there a set of windows maybe in the git download that is not in RasPI Imager version yet?  Or was I just imagining things?

Thanks for the work on this it does appear to be useful even if I have to configure it in the couple of config files.

**Comment by procount** [comment, marked as answer]
Go to the maintenance meni by pressing M twice. Here is the list of installed OSes that includes PINN at the top. From here you can edit the config.txt and/or cmdline.txt of any installed OS.

You want to highlight PINN then select the Edit Config button.
Here you can edit its config.txt and cmdline.txt files manually.
But there is a button in the bottom left of the dialog called Edit Configuration that opens up the special dialog box that you want for PINN.
